prediction/src/algorithms/segment/src/training.py

```diff
@@ -0,0 +1,88 @@
+import glob
+import math
+import os
+
+import numpy as np
+import pylidc as pl
+from keras.callbacks import ModelCheckpoint
+from keras.models import load_model
+from src.preprocess.lung_segmentation import save_lung_segments
+
+from .data_generation import get_dicom_paths, get_assets_dir
+from .model import simple_model_3d
+
+
+def get_data_shape():
+    dicom_paths = get_dicom_paths()
+    max_x, max_y, max_z = get_max_scaled_dimensions(dicom_paths)
+    return 128, 128, 128, 1  # TODO: dynamically crop shape based on maximal axes
+    return max_x, max_y, max_z, 1
```

Comment:
It's still a bit odd having two return statements. If you want to convey the same message, I'd suggest moving the first return statement to the end, and then comment out the other parts of the function. Also, could you add your above comment to the code itself so that others know why it's been commented out?

Reply:
You're right, I commented the method properly now :)

Reply:
> I'd suggest you move the first return statement to the end, and then comment out the other parts of the function.

Leaving commented-out code in PRs doesn't seem like a good practice. If there's a piece that is broken or semi-working but still merits being merged, perhaps the problems and TODOs can be tracked as by issues instead.

A `# TODO: [...]` comment used sparingly may be okay _ to explain to readers_ why something it the way it is -- it's not a helpful way to track outstanding work.

Reply:
So I should add an issue for that?

Reply:
@isms I filed #151 for that. Feel free to add comments to that issue if something is still missing.